KsApi/models/DeletePaymentMethodEnvelope.swift

```diff
@@ -17,6 +17,6 @@ extension DeletePaymentMethodEnvelope: Decodable {
     self.storedCards = try values.nestedContainer(keyedBy: CodingKeys.self, forKey: .paymentSourceDelete)
       .nestedContainer(keyedBy: CodingKeys.self, forKey: .user)
       .nestedContainer(keyedBy: CodingKeys.self, forKey: .storedCards)
-      .decode([GraphUserCreditCard.CreditCard].self, forKey: .nodes)
+      .decode([UserCreditCards.CreditCard].self, forKey: .nodes)
```

Comment:
Not sure why this is being flagged as untested, I've run the tests locally and see the coverage here.

Reply:
code coverage on xcode (Editor --> Code Coverage) shows this isn't currently being covered, but it's not a file related to this ticket, it is one of the files covered by the next ticket this sprint. I just had to update the name reference here.

Reply:
I think codecov isn't managed by xcode but rather it's website or yaml file btw